enos/modules/go-acceptance-test-target/main.tf

```diff
@@ -77,3 +79,27 @@ resource "aws_security_group" "this" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 }
+
+resource "aws_instance" "remotehost" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t3.small"
```

Comment:
Some tests were flaky because they connect to this machine. I boosted the size to give it more CPU credits.